mandatory/4-tax.js

```diff
@@ -17,7 +22,13 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+    prudactPrice= price +((price /100) *20) 
```

Comment:
const productPrice = price +((price /100) *20)
watch out for spelling.  Install the spell check extension i showed you :)